### DISCUSSION TITLE
Electribe 2 Sample Management

### DISCUSSION THREAD

**Discussion by Maxklui** [bangcorrupt/hacktribe#141] [Q&A] [answered]
@bangcorrupt 

Hello guys.

 I have a question about Electribe Sample Management. Whenever I add samples to the e2sSample.all file, I always give the samples a category, which should normally sort the samples in the Electribe. But I don't care. The kicks are in the place where I assigned them. For example, a kick is on slot 530 and another kick is on slot 515. Although I've put this sample in the kicks category as well. Does anyone know what the error is or what I'm doing wrong or who knows the solution from you Because with other Hacktribe users it's also sorted. Many thanks for the help Best wishes

**Comment by bangcorrupt** [comment, marked as answer]
> I always give the samples a category, which should normally sort the samples in the Electribe.

Does this normally move the samples?  I don't think it does.  

Holding Shift and scrolling the Oscillator encoder should normally skip to the next category, but this is currently broken #13.

**Reply by Maxklui** [reply]
@bangcorrupt

I mean, the samples aren't even sorted when I import them. That's the problem.

**Reply by bangcorrupt** [reply]
They won't be sorted automatically.  Categories are just tags.

**Reply by maxtribe** [reply]
@bangcorrupt 

Is there a way that I can get all my samples sorted on the Electribe? Thank you in advance

**Reply by bangcorrupt** [reply]
Sort them in the sample manager.
